### ISSUE TITLE
Soil temperature simulation fails on unit test/example inputs

### ISSUE THREAD

**Issue by dschlaep** [DrylandEcology/rSOILWAT2#90]
From `tests/testthat/test_exec_and_aggregate.R`
```
it <- "Ex2"
sw_input <- readRDS(paste0(it, "_input.rds"))
rd <- sw_exec(inputData = sw_input, weatherList = sw_weather, echo = FALSE,
      quiet = TRUE)
expect_false(has_soilTemp_failed())
```

> Error: has_soilTemp_failed() isn't false.

**Comment by dschlaep** [comment]
Now `it = "Ex3`.

Once fixed, remove special treatment in `test_exec_and_aggregate.R`:
```
    # This doesn't work for Ex3 'soil temperature' (see issue #90 'Soil temperature simulation fails on unit test/example inputs')
    if (it != "Ex3") {
        expect_silent(sw_exec(...
```

**Comment by dschlaep** [comment]
closed and tested with commit d2b5fb926e8d1522362088ec488599864e192394
